README.md

```diff
@@ -25,7 +25,7 @@ Stay tuned for more tasks!
 # Important Notes
 
 - If you have any questions, suggestions, or proposals please use the support portal:(https://support.concordium.software/c/bounty-program/14).
-- If you are going to start building a bounty task, please comment on the issue so that others can track it.
+- If you are going to start building a bounty task, please comment on the issue so that others can track it. If you declared and submitted anything, or haven't shown any progress for a long time Concordium team has a right to give the task to any other volunteer. From that moment first come first serve will apply. 
```

Comment:
```suggestion
- If you are going to start building a bounty task, please comment on the issue so that others can track it. If you commented on the issue but haven't submitted anything, or haven't shown any progress for a long time Concordium team has a right to give the task to any other volunteer. From that moment first come first serve will apply. 
```